### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by pgrete**
## PR Summary

As discussed during the last call it's time for the next release.
I'll add the tag and notes once this is merged so that it can point at the right commit.

### CODE REVIEW

**Review by JoshuaSBrown — APPROVED** [review]

**Comment by pgrete** [comment]
Ping. We need a second approval before we can merge.

**Review by jlippuner — APPROVED** [review]

**Pull request merged**
